pkgs/tools/misc/terminal_tarot/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "terminal_tarot";
+  version = "v0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "aswan89";
+    repo = pname;
+    rev = version;
+    sha256 = "0s1ahv7904z7z9r062pfgx3dn9z31l4xxq1vpizib969mvvnfg13";
+  };
+
+  cargoSha256 = "1djdm8b4q4s17kvn6392qy1zgm4jz14jzijsc4z6xrrn4mhpayvc";
+
+  meta = with stdenv.lib; {
+    description = "Toy application for making tarot readings at the terminal, Written in Rust";
+    homepage = "https://github.com/aswan89/terminal_tarot";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Only;
```

The gpl license variants without the `Plus` or `Only` suffix are deprecated, because they do not specify the license well enough. Use `Plus` if the project has "or later" sections in the actual source code files (not the license). Otherwise, use the `Only` suffix.
 
See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.